config/locales/views/criteria/es.yml

```diff
@@ -15,19 +15,22 @@ es:
     ordering_criteria: "Ordenando los criterios de calificación"
     help_editing_criteria_message: "Seleccione un criterio de calificación en la izquierda para editarlo.
       Cambios para los diferentes criterios de calificación pueden ser realizados de la siguiente manera:"
-    help_editing_rubric_criteria_message: "Los nombres de los criterios de calificación, encabezados de nivel,
+    help_editing_rubric_criteria_message:
+      "Los nombres de los criterios de calificación, encabezados de nivel,
```

Comment:
Revert the newline changes in this file (they are not related to your pull request).